src/App.jsx

```diff
@@ -14,70 +15,77 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [selected, setSelected] = useState('Jam');
+  const selectGood = good => setSelected(good);
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+  return (
+    <main className="section container">
+      {
+        selected
+          ? (
+            <h1 className="title is-flex is-align-items-center">
+              { `${selected} is selected`}
 
-      <button
-        data-cy="ClearButton"
-        type="button"
-        className="delete ml-3"
-      />
-    </h1>
+              <button
+                data-cy="ClearButton"
+                type="button"
+                className="delete ml-3"
+                onClick={() => selectGood('')}
```

Comment:
```suggestion
                onClick={clearGood}
```
I'd recommend creating separate functions to handle events. This way your code will be more maintainable, and easier to extend and re-use.